apps/docs/docs/state/integrations/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
   "label": "Integrations",
-  "position": 4,
+  "position": 90,
```

Comment:
Why are you changing the sidebar position to x10 number?
Did you mean 9?

Reply:
This is the second-to-last item.